docs/api.rst

```diff
@@ -9053,7 +9053,7 @@ Adding blockchain accounts
 
       This endpoint can also be queried asynchronously by using ``"async_query": true``
 
```

Comment:
The documentation when an account is removed should be also be updated

Reply:
M? What's changed?

Reply:
You now are returning the balances right? This should be in the response at the docs

Reply:
M? What's changed?

Reply:
Well, I am returning balances but I haven't changed docs in the balances query removal pr :smiley:, so it is the same as before that pr (and thus says that balances update is returned)